orchestration/dagster_orchestration/hca_orchestration/tests/test_pipelines.py

```diff
@@ -43,7 +43,7 @@ def run_pipeline(
             mode=pipeline_mode
         )
 
-    @pytest.mark.e2e
+    @pytest.mark.skip
```

Comment:
you should be able to layer on multiple markers rather than swapping the e2e marker out, right?